### ISSUE TITLE
custom android 13 icon theming

### ISSUE THREAD

**Issue by RustoMCSpit** [ShacharHarshuv/open-ear#228]

**Comment by ShacharHarshuv** [comment]
@RustoMCSpit Can you please add more information to this ticket? Otherwise I'll have to close it for lack of info

**Comment by RustoMCSpit** [comment]
> @RustoMCSpit Can you please add more information to this ticket? Otherwise I'll have to close it for lack of info

https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

**Comment by RustoMCSpit** [comment]
android 13 allows for monochrome icons as well material you designing

**Comment by ShacharHarshuv** [comment]
I don't understand what you mean. What is actually the requirement here?

**Comment by RustoMCSpit** [comment]
> I don't understand what you mean. What is actually the requirement here?

to give the app itself the ability to adopt a monochrome theme, i can screenshot my homescreen if you want?

**Comment by ShacharHarshuv** [comment]
I doubt that anyone will prioritize this, but you are welcome to figure out how this can be done and create a PR.

**Comment by RustoMCSpit** [comment]
> I doubt that anyone will prioritize this, but you are welcome to figure out how this can be done and create a PR.

sure but do you understand what im saying at leasr

**Comment by ShacharHarshuv** [comment]
> sure but do you understand what im saying at leasr

Yes, but I'm not familiar with this feature, and believe there is little request for it.

**Comment by RustoMCSpit** [comment]
> > sure but do you understand what im saying at leasr
> 
> Yes, but I'm not familiar with this feature, and believe there is little request for it.

with android 13 most apps support it, i'll look into pulling

**Comment by ShacharHarshuv** [comment]
It doesn't feel like it has much value. Closing the issue, but feel free to open a PR if you really want this.
